src/core/api/instance_api.cpp

```diff
@@ -135,7 +135,7 @@ const char *otGetVersionString(void)
 
 #if !defined(OPENTHREAD_BUILD_DATETIME) && defined(__ANDROID__)
 
-#ifdef OPENTHREAD_ENABLE_ANDROID_NDK
+#ifdef OPENTHREAD_CONFIG_ANDROID_NDK_ENABLE
```

Comment:
We need to add a macro name check `openthread-core-config-check.h` so that all users who are still using `OPENTHREAD_ENABLE_ANDROID_NDK` will get a compile error when updated to the latest commit.

Reply:
Nice! Thanks @wgtdkp for noticing this. 👍